ViAn/Filehandler/filehandler.h

```diff
@@ -15,93 +15,107 @@
 #include "project.h"
 #include "dir.h"
 #include <algorithm>
-
-
-
+#include <QJsonObject>
+#include <QFile>
+#include <QDir>
+#include <QTextStream>
+#include <QJsonDocument>
```

Comment:
Please seperate the different things with an empty line. At least there should be one after all the includes.